site/content/contribute/server/developer-workflow.md

```diff
@@ -28,7 +28,7 @@ Here's a general workflow a Mattermost developer working on the [mattermost-serv
      * Run individual tests by name executing `go test -run "TestName"`.
      * Run all the tests in a package where changes were made executing `go test app`.
      * Create a draft PR with your changes and let our CI servers run the tests for you.
-7. Commit your changes, push your branch and [create a pull request](https://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request).
+7. Commit your changes, push your branch and [create a pull request](https://developers.mattermost.com/contribute/getting-started/).
```

Comment:
In fact, this is a better page to link to

```suggestion
7. Commit your changes, push your branch and [create a pull request](https://developers.mattermost.com/contribute/getting-started/contribution-checklist/).
```

Reply:
Yeah I will make a commit ASAP. Thanks for responding :D